uktrade/data-hub-frontend
# Removed the banner announcing export wins moving to DataHub

## Description of change

This PR removes the banner announcing that Export Wins are moving to DataHub.

## Test instructions

1. Go to `/`
2. You should no longer see the "Important: Export Wins is moving to Data Hub shortly" banner underneath the main navigation

## Screenshots

### Before

![banner-before](https://github.com/uktrade/data-hub-frontend/assets/2333157/7f9ad0f1-1a00-46c9-852d-748d76bc6d29)

### After

![banner-after](https://github.com/uktrade/data-hub-frontend/assets/2333157/39e5bdd1-4b91-477c-bdd8-bd09517c136b)
